projects/Mallard/src/components/spinner.tsx

```diff
@@ -10,7 +10,7 @@ const styles = StyleSheet.create({
         margin: 2,
         borderRadius: 100,
     },
-    container: { flexDirection: 'row' },
+    container: { flexDirection: 'row', padding: 5 },
```

Comment:
The vertical animation on the new loader makes the dots escape their view box so I've added this padding here to stop this from happening (it was overlapping onto the header when signing in).